### ISSUE TITLE
Libraries/Playlists not loading

### ISSUE THREAD

**Issue by dramirvf** [ukdtom/ExportTools.bundle#133]
Hi,
**PMS Version:** PlexMediaServer-1.19.3.2843-e3c1f7bcd
**Server:** Synology NAS (DSM 6.2.2-24922 Update 4)

I have installed the latest version of **ExportTools.bundle**.

Export path has been specified correctly.
Full permissions have been given to Plex Account.

Unfortunately Libraries/Playlists won't load at all.
I have tried changing the configuration setting of ExportTools.bundle. Nothing loads.

Btw a folder by the name of **ExportTools** has been automatically generated in the specified path. It looks that the plugin have detected and has access to that folder. But Libraries/Playlist do not load.

Any solutions?

**Comment by ukdtom** [comment]
The logs you provided are Plex logs, and those doesn't include 3.Party

See: https://github.com/ukdtom/ExportTools.bundle/wiki/Trouble-Shooting

**Comment by dramirvf** [comment]
I attached the relevant logs here.
[com.plexapp.plugins.ExportTools.zip](https://github.com/ukdtom/ExportTools.bundle/files/4664587/com.plexapp.plugins.ExportTools.zip)

**Comment by ukdtom** [comment]
Please try this build:
https://github.com/ukdtom/ExportTools.bundle/archive/tommy-work.zip

And regardless of working or not, I need a new set of logs please

**Comment by dramirvf** [comment]
I tried the new build.
It didn't work. Exact same behavior.

Here are the new logs.
[com.plexapp.plugins.ExportTools.log_22-05-20-16-28.zip](https://github.com/ukdtom/ExportTools.bundle/files/4667867/com.plexapp.plugins.ExportTools.log_22-05-20-16-28.zip)

**Comment by ukdtom** [comment]
I didn't expect it to, since all it did, was adding additional logging ;)

From what I can tell, some settings you entered is invalid?

Could you do a hard reset of the plugin...

Wrote a HowTo for WebTools, and same procedure:
https://github.com/ukdtom/WebTools.bundle/wiki/Reset#manually

**Comment by dramirvf** [comment]
All settings are the default ones except for the path which I have added.

How could I hard reset plugin through SSH?

Information you have provided is reset through Web Tools. I don't use 
webtools. I have installed the plugin manually through SSH and I used the 
graphical interface through plex web for settings.

**Comment by ukdtom** [comment]
Follow the procedure, and when you read the word WebTools, replace it with ExportTools

**Comment by dramirvf** [comment]
I did the hard reset as instructed and then installed the latest plugin build you gave me. Still not working. The same behavior.

No library or playlist will be loaded.

**I have included my settings here:**
![settings](https://user-images.githubusercontent.com/40552131/82708116-c1df0c00-9c92-11ea-818f-e1a990c8eb28.jpg)

**And here is my latest logs:**
[com.plexapp.plugins.ExportTools_23-05-20-01-05-19.zip](https://github.com/ukdtom/ExportTools.bundle/files/4670303/com.plexapp.plugins.ExportTools_23-05-20-01-05-19.zip)

**All permissions (Full Control) have also been given to Plex account and Admin account for the given path and libraries.**

**Comment by ukdtom** [comment]
Please apply the following:

https://github.com/ukdtom/ExportTools.bundle/archive/tommy-work.zip

It contains additional logging, and no fix, since still haven't figured out what goes wrong here :(

Also, did you tell ET to pull your Library and Playlist items from the top two dropdown boxes?

**Comment by dramirvf** [comment]
This bundle does not bring the ET in the plugin lists!!!
So I can't test it and make logs.

I put the drop-down to **"Load Library List"/"Load Playlists"**, then save the setting. Then re-open the settings and still no library/playlist is listed.

I've tried to load them one by one or both. No combination results in the appearance of libraries/playlists in the drop-down.

**Comment by ukdtom** [comment]
This bundle does not bring the ET in the plugin lists!!!
Cool, and why I added additionally logging to that point!

So please provide the logs

**Comment by dramirvf** [comment]
Sorry. I though it should bring ET up to generate logs.

Here are the latest logs.
[com.plexapp.plugins.ExportTools.log_23-05-20-09-58-58.zip](https://github.com/ukdtom/ExportTools.bundle/files/4671200/com.plexapp.plugins.ExportTools.log_23-05-20-09-58-58.zip)

**Comment by ukdtom** [comment]
Darn, I forgot one stupid comma, make the plugin stop :(

So sorry, and if you could try again, and afterwards provide logs
https://github.com/ukdtom/ExportTools.bundle/archive/tommy-work.zip

**Comment by dramirvf** [comment]
The plugin appeared in the plugin list this time!

I tried several times so it generate logs.

Here are the logs:
[com.plexapp.plugins.ExportTools.log_23-05-20-16-56-02.zip](https://github.com/ukdtom/ExportTools.bundle/files/4671744/com.plexapp.plugins.ExportTools.log_23-05-20-16-56-02.zip)

**Comment by ukdtom** [comment]
And with the new set of logs, we found the error ;)

`2020-05-23 16:54:43,952 (7fba97159700) :  DEBUG (__init__:192) - Exception handling due to [Errno 13] Permission denied: '/volume1/Plex/Library/Application Support/Plex Media Server/Plug-ins/ExportTools.bundle/Contents/DefaultPrefs.json'`

You need to give the plex user full rights to the ExportTools.bundle directory

**Comment by dramirvf** [comment]
Thanks. It solved the issue.
Plex account had the full access to everything in the Plex folder except this plug-in. The reason was that I copied files through SSH while I had logged in with root account. So root was the owner without giving any access to others!

The plugin works but there is something happening at the beginning while saving the settings.
An error message pops up!

Here is the snapshot:
![image](https://user-images.githubusercontent.com/40552131/82735760-51d79100-9d39-11ea-823f-1aab7ef5d27a.png)

Although it gives error, the plugin works.

This error message pops up with both versions (V2.0.0.10 & V2.0.0.11 which you customized for maximum logs.

I have attached the extensive logs here.
[com.plexapp.plugins.ExportTools.log_23-05-20-21-01-35.zip](https://github.com/ukdtom/ExportTools.bundle/files/4672129/com.plexapp.plugins.ExportTools.log_23-05-20-21-01-35.zip)
